src/olympia/landfill/management/commands/generate_addons.py

```diff
@@ -38,9 +38,9 @@ class Command(BaseCommand):
     )
 
     def handle(self, *args, **kwargs):
-        if not settings.DEBUG:
-            raise CommandError('You can only run this command with your '
-                               'DEBUG setting set to True.')
+        # if not settings.DEBUG:
+        #    raise CommandError('You can only run this command with your '
+        #                       'DEBUG setting set to True.')
```

Comment:
@EnTeQuAk said he would work on something for this. I'll add a note that it must be re-added.